source/TextArea.h

```diff
@@ -0,0 +1,76 @@
+/* TextArea.h
+Copyright (c) 2024 by thewierdnut
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef TEXTAREA_H
+#define TEXTAREA_H
```

Comment:
```suggestion
#ifndef TEXT_AREA_H_
#define TEXT_AREA_H_
```
(according to the [style guide](https://endless-sky.github.io/styleguide/styleguide.xml#Files))